README.md

```diff
@@ -1 +1,5 @@
-# Review
\ No newline at end of file
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
```

Comment:
Nice job! but if you provide information about your project in readme file it would be nice, because readme contain information about your project 😄 👍 💯